src/components/picture/__tests__/CdrPicture.spec.js

```diff
@@ -0,0 +1,23 @@
+import { shallowMount } from '../../../../test/vue-jest-style-workaround.js';
+import CdrPicture from '../CdrPicture.vue';
+
+describe('CdrPicture', () => {
+  it('matches snapshot', () => {
+    const wrapper = shallowMount(CdrPicture, {
+      props: {
```

Comment:
thoughts on how palette could change the image source? Atlassian has and interesting take 
https://atlassian.design/components/image/examples#dark-mode